netsuite/11.3/modules/ROOT/pages/netsuite-examples.adoc

```diff
@@ -0,0 +1,751 @@
+		<foreach doc:name="For Each">
+			<ee:transform doc:name="Transform to JSON">
+			<ee:message>
+				<ee:set-payload><![CDATA[%dw 2.0
+output application/json
+ns ns0 urn:core_2020_2.platform.webservices.netsuite.com
+---
+payload.ns0#record
+]]></ee:set-payload>
+			</ee:message>
+		</ee:transform>
+			<logger level="INFO" doc:name="Logger" message="#[payload]" />
+		</foreach>
+	</flow>
+</mule>
+----
+
+== Custom Records and Fields
+
+The following examples show how to add, get, and delete custom records with
+custom fields.
+
+[[add-custom-record]]
+=== Add a Custom Record
+
+This example shows how to add a custom record with custom fields using the *Add*
+operation. This example is generated using DataSense.
+
+Pick a recordType in the operation to generate metadata. In this example, the
+key `\\__customRecordType__customrecord_2663_bank_details\__374` is used. Keys consist of
+*separators*, *recordType*, *scriptId*, and *internalId*. For this example, the
+separator is `__`, the recordType is `customRecordType`, the scriptId is
+`customrecord_2663_bank_details`, and the internalId is `374`.
+
+The following code shows a DataWeave example for adding a custom record with
+custom fields:
+
+----
+%dw 2.0
+output application/xml
+ns ns0 urn:messages_2020_2.platform.webservices.netsuite.com
+ns ns01 urn:customization_2020_2.setup.webservices.netsuite.com
+ns ns02 urn:core_2020_2.platform.webservices.netsuite.com
+ns xsi http://www.w3.org/2001/XMLSchema-instance
+---
+{
+	ns0#add: {
+		ns0#record @("xmlns:ns01": ns01, xsi#"type": "ns01:CustomRecord"): {
+			ns01#customFieldList: {
+				StringCustomFieldRef__custrecord_2663_legal_name__3878: {
+					ns02#value: "test"
+				},
+				LongCustomFieldRef__custrecord_2663_eft_file_cabinet_id__3874: {
+					ns02#value: 1234
+				},
+				BooleanCustomFieldRef__custrecord_2663_dd_batch__3864: {
+					ns02#value: true
+				}
+			}
+		}
+	}
+}
+----
+
+The customFieldList contains the customFields for the entity you use.
+
+The example above shows `StringCustomFieldRef\__custrecord_2663_legal_name__3878`, `BooleanCustomFieldRef\__custrecord_2663_dd_batch__3864` and
+`LongCustomFieldRef\__custrecord_2663_eft_file_cabinet_id__3874`. The field names consist of
+a *custom field ref type*, *scriptId*, and *internalId*.
+
+`StringCustomFieldRef\__custrecord_2663_legal_name__3878` also has a specified value
+`ns02#value: "test"`,  `LongCustomFieldRef\__custrecord_2663_eft_file_cabinet_id__3874` has a specified
+value `ns02#value: 1234`, and `BooleanCustomFieldRef\__custrecord_2663_dd_batch__3864` has a specified value `ns02#value: true`.
+
+[[get-custom-record]]
+=== Get Custom Record
+
+This example shows how to get a custom record with custom fields using the *Get*
+operation. This example is generated using DataSense.
+
+Include the following attributes to get a custom record:
```

Comment:
The indentation of the list continue text in Preview shows that it is not lined up under the bullet. I think you need to use list continuation characters (+).